client/src/components/Common/Buttons/PrimaryButton/index.js

```diff
@@ -1,26 +1,25 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { theme1, theme2 } from "../../../../resources/style-constants";
+import { theme1, theme2, lightBlue, grey3 } from "../../../../resources/style-constants";
 import Button from "../Button";
 
+
 const theme = {
   default: theme1,
   hover: theme2,
   disabled: theme1
 };
 
-const PrimaryButton = ({
-  onClick,
-  fullWidth,
-  disabled = true,
-  children,
-  ...props
-}) => {
-  return (
-    <Button {...{ fullWidth, theme, disabled, onClick, ...props }}>
-      {children}
-    </Button>
-  );
+const accent = {
```

Comment:
I needed these changes for the buttons, so I copied over your code and modified it a little more. You'll want to revert your changes to this file, since the updated version is already in master.